tools/plots.py

```diff
@@ -142,7 +155,7 @@ def plotdir(run_dir, platform):
         logging.info('No performance data found')
 
     # Load Trace Analysis modules
-    trace = Trace(platform, run_dir)
+    trace = Trace(platform, run_dir, [], trace_format="systrace")
```

Comment:
Just checking why `trace_format` should be systrace. tools/plot.py should should be generic, not android-specific right?

Reply:
Fixed. libs.utils.analysis.Trace loads systrace and ftrace a bit inconsistently (the distinction between file and directory isn't clear). New code will detect systrace and call Trace appropriately.

Reply:
Hmm, so it looks like the new commit isn't updated in this PR.  As per "https://github.com/ARM-software/lisa/pull/554/commits", this commit (f8082d6702d) is 21 days old. Could you update it?